service/spi/src/main/java/org/kaazing/gateway/service/ServiceProperties.java

```diff
@@ -44,6 +44,8 @@
  */
 public interface ServiceProperties {
 
+    public static final String LIST_SEPARATOR = ",";
```

Comment:
This LIST_SEPARATOR constant is an implementation detail of the implementation of TurnRestService. Another class might use another LIST_SEPARATOR. This should be moved in TurnRestService